PowerShell Scanners/Windows Firewall Rules/Windows Firewall Rules.ps1

```diff
@@ -0,0 +1,19 @@
+[cmdletBinding()]
+param(
+    [parameter()]
+    [switch]
+    $Enabled
+)
+
+process{
+    
+    if([version](Get-WmiObject win32_operatingsystem).Version -lt 6.3.0){
```

Comment:
The PowerShell Scanner is only compatible with PowerShell 3+, so you can use `Get-CimInstance` if you want.

Reply:
Fixed!